app_dart/lib/src/foundation/utils.dart

```diff
@@ -103,6 +103,22 @@ Future<List<Map<String, dynamic>>> getRepoBuilders(HttpClientProvider branchHttp
       .toList();
 }
 
+/// Returns [builder] when any file falls in any pre-set dir.
```

Comment:
Update? given that functions in this library are made for reusability let's add complete docs. E.g parameters received, return value and an example.

Reply:
Updated.